demopack/demo_sso/home.html

```diff
@@ -58,13 +58,14 @@
     <!-- Bootstrap core JavaScript
     ================================================== -->
     <!-- Placed at the end of the document so the pages load faster -->
-    <script src="./js/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+    <script type="text/javascript" src="./js/jquery-2.1.1.min.js"></script>
```

Comment:
Originally jquery 3.2.1 was used. why you downgraded the version?

Reply:
As slim version was not supporting ajax call so i replaced it with same min version which was already used in application.